analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DoNotDecreaseMemberVisibility.CSharp11.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
Lets have a call tomorrow to go into the details, both DoNotDecreaseMemberVisibility and DoNotHideBaseClassMethods test cases are not valid.

Reply:
Sorry for the confusion. They make sense!